keps/0002-controller-manager.md

```diff
@@ -257,6 +263,102 @@ In case of the cloud-controller-manager, the deployment should be deleted using
 kubectl delete -f cloud-controller-manager.yml
 ```
 
+### Implementation Details/Notes/Constraints
+
+#### Repository Requirements
+
+**This is a proposed structure, and may change during the 1.11 release cycle.
+WG-Cloud-Provider will work with individual sigs to refine these requirements
+to maintain consistency while meeting the technical needs of the provider
+maintainers**
+
+Each cloud provider hosted within the `kubernetes` organization shall have a
+single repository named `kubernetes/cloud-provider-<provider_name>`. Those
+repositories shall have the following structure:
+
+* A `cloud-controller-manager` subdirectory that contains the implementation
+  of the provider-specific cloud controller.
+* A `docs` subdirectory.
+* A `docs/cloud-controller-manager.md` file that describes the options and
+  usage of the cloud controller manager code.
+* A `docs/testing.md` file that describes how the provider code is tested.
+* A `Makefile` with a `test` entrypoint to run the provider tests.
```

Comment:
Shall this be enforced? This should be described in 'docs/testing.md' right?

Reply:
Yeah, I think we may want to change this in the future, a lot of what we have now is subject to change and more of a starting point for us to iterate on :)